src/components/SRNEntry.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <section>
+    <div v-if="plioUse" class="flex flex-col mt-8">
```

Comment:
Not `plioUse`. This component ka code shouldn't know about use case. this `v-if` should just be `ifSingleSRNonly` or something like that

Reply:
I refactored it like Deepansh said. Used a computed property `isRedirectToPlio`. It computes the value based on the `redirectTo` param.